components/BlogIcons.tsx

```
@@ -0,0 +1,33 @@
+    strokeLinejoin="round"
+  >
+    <rect width="14" height="14" x="8" y="8" rx="2" ry="2" />
+    <path d="m4 16c-1.1 0-2-.9-2-2V4c0-1.1.9-2 2-2h10c1.1 0 2 .9 2 2" />
+  </svg>
+)
+
+// Checkmark Icon - Fill style (preferred from test)
+export const CheckmarkIcon: React.FC<BlogIconProps> = ({
+  className = 'w-4 h-4',
+}) => (
+  <svg viewBox="0 0 24 24" className={className} fill="currentColor">
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)